src/components/list/Item.js

```diff
@@ -22,9 +22,13 @@ export default function Item({ data }) {
           )}
           {data.icon && (
             <FontAwesomeIcon
-              icon={socialIcon(data.icon)}
+              icon={socialIcon(data.icon).icon}
               className="h-12 w-12 flex-none rounded-full bg-gray-50 p-2"
-              style={{ height: 50, width: 50 }}
+              style={{
+                height: 50,
+                width: 50,
+                color: `${socialIcon(data.icon).color}`,
```

Comment:
Does this need to have the `? Would this work...

```suggestion
                color: socialIcon(data.icon).color,
```

Reply:
I have removed the template literal

Reply:
Great 👍